Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -94,9 +94,10 @@ static NugetHelper()
             {
                 return;
             }
-            
+
 #if UNITY_5_6_OR_NEWER
             DotNetVersion = PlayerSettings.GetApiCompatibilityLevel(EditorUserBuildSettings.selectedBuildTargetGroup);
+            Debug.LogWarning(string.Format("The project's API version is {0} attention please the API version supproted you want to download the nuget package.", DotNetVersion));
```

Comment:
I'm not sure why we'd always want a warning for older versions of Unity?

Reply:
Thanks for noticing me. It's fixed already.